### ISSUE TITLE
config.json started mutating during normal use

### ISSUE THREAD

**Issue by srhb** [doy/rbw#74]
This breaks on eg. home-manager and anything else where we're provisioning a declarative config which is not writable.

```
rbw get: failed to save config to /home/sarah/.config/rbw/config.json: Read-only file system (os error 30)
```

See: https://github.com/nix-community/home-manager/issues/2476

**Comment by rycee** [comment]
Seems like [7b95f7b9d84c44ca273c1a1ce7ab62925d4a5def](https://github.com/doy/rbw/commit/7b95f7b9d84c44ca273c1a1ce7ab62925d4a5def#diff-cba64c21ab992eaad29fce147a08f4560a4769bc14682b8a96081a5fd02dbecdR136) contains the problematic line. Perhaps it would be possible to add a condition in the `if` statement to verify that the file is writable?

**Comment by doy** [comment]
fixed in 1.4.2
